src/components/App.js

```diff
@@ -7,7 +7,7 @@ import '../css/App.css';
 
 const stats = require('../stats.json');
 const years = Object.keys(stats).map(k => k.substring(0, 4)).reduce((p, c) => {
-  if (!p.includes(c)) p.push(c);
+  if (p.indexOf(c) === -1) p.push(c);
```

Comment:
Odd! [react-scripts](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/package.json) includes the [core-js](https://github.com/zloirock/core-js) ES6 polyfills which I assumed would cover `includes`.